roles/monitoring_plugins/tasks/install_on_RedHat.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Yum - install epel-release
+  become: yes
+  yum:
+    state: present
+    name: "epel-release"
+    update_cache: true
+  when: icinga_monitoring_plugins_epel
```

Comment:
Is it possible to reference our collection repo role and let it install epel? 

They won't interfere with each other but would be really clean?

Reply:
I think it would be possible if we changed the repos role to skip tasks if certain variables are undefined.  
This however would defeat the purpose of *"wanted failures"*.  

If I were to include the repo role in this new role (ignoring variables), the icinga repositories would be deployed/activated on the hosts, even though the user didn't directly run/include `icinga.icinga.repos`.  

I see 3 possible solutions:  

1. Keep it as is. Let this new role enable epel.  
2. Expect the user to somehow activate epel. Don't enable it in this role. Fail when trying to install.  
   Possibly use an assertion (`shell: yum repolist enabled`)  
3. Add a lot of conditionals to the repos role in order to skip tasks if certain variables aren't set to true (e.g. `icinga_repo_add_icinga_repo_release: false`).  
   This would however require the user to explicitly set those variables (if their default is not `true`).  
   Side note: As a by-product users could then decide whether they want to install `icinga-release`, `icinga-testing` and/or `icinga-snapshot`. 

I'm open to any other suggestions but would probably prefer option 2.

Reply:
Ah @Donien you got me wrong I guess, what I meant was the handling is right. The user should be able to handle the epel at the role usage. But instead of a task installing epel, we could refer to our role and "use the role" in a task. 

Example: 
```
- name: Manage epel repo for purpose
   include_role:
     name: repos
     vars:
        icinga_repo_epel: true
        icinga_repo_stable: false
        icinga_repo_testing: false
        icinga_repo_snapshot: false
   when: icinga_monitoring_plugins_epel
```

Sorry as I was busy could respond in time.

Reply:
The task has been changed.

```
- name: Activate epel repository
  include_role:
    name: icinga.icinga.repos
  vars:
    icinga_repo_epel: true
    icinga_repo_stable: false
    icinga_repo_testing: false
    icinga_repo_snapshot: false
  when: icinga_monitoring_plugins_epel
```